app/models/projects/activity.rb

```diff
@@ -33,13 +33,12 @@ def self.included(base)
 
   module ActivityScopes
     def latest_project_activity
-      @latest_project_activity ||=
```

Comment:
I am not 100% certain yet where this is happening, but when we constantize `Queries::Projects::ProjectQuery` early (as we do by adding it to the allowed entitiesin members), this method is executed before all initializers have run andall providers are registered. This way when loading activity it does not check any of the additional activity providers (i.e. meetings) and a bunch of tests fail